Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM golang:1.23-alpine AS builder
```

Comment:
use this simpler dockerfile allows for easier testing

```
FROM debian:stable-slim

WORKDIR /app

COPY goback /app/goback
COPY test_files/ /app/test_files/

RUN apt-get update
RUN apt upgrade -y
RUN apt-get install -y openssh-client

RUN mkdir /app/private
RUN mkdir /app/.data

RUN ssh-keygen -t rsa -b 4096 -f /app/private/id_rsa -N ""

CMD ["goback"]
```

Reply:
Don't use debian Alpine is for faster and smaller container builds

Reply:
Make directory using single command also this won't register changes whenever code is changed bc how Docker handles caching.

Reply:
> Make directory using single command also this won't register changes whenever code is changed bc how Docker handles caching.

??

Reply:
> Don't use debian Alpine is for faster and smaller container builds

doesn't really matter, we can use alphine
this is mainly for testing right?

Reply:
Receiver ki same IP hai dono me?

Reply:
![image](https://github.com/user-attachments/assets/b9159c45-2fa1-40e1-ba35-c31531f019a8)

![image](https://github.com/user-attachments/assets/3edd650d-b9e7-4dba-8a7d-b9b71d5b50de)


nhi alag alag hai

Reply:
pata nhi
samajh nhi aa rha